Week3/homework/step2-7.js

```diff
@@ -21,3 +21,11 @@ f2(y);
 console.log(y);
 
 // Add your explanation as a comment here
+/* Primitive variables can not be changed by function, but array or object can be changed. Because when we write array or object
+as a argument, it is approached with its reference/address. And when we change object in  function, we are changing 
+the referenced/original object. Dont ask why, it is Javascript : )
```

Comment:
But we are asking why! I think you get the general idea, try to explain a bit more by using the code from the task.

Reply:
I added more explanation using codes as much as possible. Thanks for review Paul

Reply:
👍